content/intro-to-storybook/vue/fr/simple-component.md

```diff
@@ -0,0 +1,281 @@
+---
+title: 'Construire un simple composant'
+tocTitle: 'Un simple composant'
+description: 'Construire un simple composant en isolation'
+commit: b2274bd
+---
+
+Nous allons construire notre propre interface utilisateur en suivant la méthode [Component-Driven Development](https://blog.hichroma.com/component-driven-development-ce1109d56c8e). C'est un processus qui créé des interfaces utilisateurs du début à la fin, en commençant par les composants et terminant avec les écrans. CDD vous aide à adapter la complexité à laquelle vous êtes confronté lors de la création de l'interface utilisateur.
```

Comment:
@jeanphibaconnais when you're able please adjust the link "Component-Driven Development" to point at https://www.componentdriven.org/

Reply:
it's modified 😄